app/models/user.rb

```diff
@@ -5,5 +5,5 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-
+  validates :last_name, presence: true
```

Comment:
Is there a reason why we have a validation for last_name here?

Reply:
I put it there because of our db schema...